package.json

```diff
@@ -1,30 +1,35 @@
 {
   "name": "iexec-voucher-subgraph",
-  "version": "0.1.0",
+  "version": "1.0.0",
   "description": "Subgraph for iExec Voucher contracts",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/iExecBlockchainComputing/iexec-voucher-subgraph.git"
   },
-  "license": "Apache-2.0",
+  "license": "ISC",
   "bugs": {
     "url": "https://github.com/iExecBlockchainComputing/iexec-voucher-subgraph/issues"
   },
   "homepage": "https://github.com/iExecBlockchainComputing/iexec-voucher-subgraph#readme",
+  "main": "index.js",
   "scripts": {
     "schema": "./scripts/generate-schema-and-subgraph.sh",
-    "codegen": "graph codegen",
+    "codegen": "cp subgraph.template.yaml subgraph.yaml && graph codegen",
```

Comment:
Why not renaming subgraph.template.yaml to subgraph.yaml instead?

Reply:
I'm going to create a script that will generate our needed subraph.yaml from this, so it will indeed play the role of a template for several networks